### PR TITLE
Update 04.2022.1 - WCA Board

### PR DESCRIPTION
**Pull request by Epride**
Removal of section 7 which was contradicted the requirements of the Bylaws

### CODE REVIEW

File: documents/motions/04.2022.1 - WCA Board.md
Comment (KerrieJarman) [inline documents/motions/04.2022.1 - WCA Board.md:1]: These seem sensible to me

File: documents/motions/04.2022.1 - WCA Board.md [hunk 1]
```diff
@@ -42,11 +44,5 @@ The WCA Board of Directors is the leadership team of the WCA and its highest aut
...
-   1. WCA Voting Members can request a voting to the WCA Ethics Committee to remove all the WCA Board according to Bylaws 3.8 (Removal).
-      1. In order to consider such a request, the request must be signed by at least 20% of the WCA Voting Members.
-      2. The WCA Ethics Committee will lead the process of the voting that will take place.
-   2. A Supermajority among all WCA Voting Members is required to remove all the WCA Board.
-      1. If the removal is accepted, the WCA Ethics Committee will start an election of new Directors immediately. During the election period, duties and rights of the WCA Board will not be operational and thus the processes that are led by WCA Board will not be possible.
-8. Reducing conflicts of roles
-   1. There may be no overlap in members of the WCA Board, WCA Disciplinary Committee, and WCA Ethics Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader.
+7. Reducing conflicts of roles
+   1. Members of the WCA Board must not be members of the WCA Disciplinary Committee or WCA Ethics Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader
```
Comment (EdHollingdale) [inline documents/motions/04.2022.1 - WCA Board.md]: Need to rephrase given other changes

**Comment by lgarron** [comment]
> Removal of section 7 which was contradicted the requirements of the Bylaws

Would it be possible to elaborate exactly why that is, and why not to take another path (like amending the Bylaw)? The "vote of no confidence" is in place as a "break glass" mechanism in lieu of more complicated checks and balances. Without also introducing specific mechanisms to compensate, I'd be worried about the long-term danger of removing section 7.

**Comment by mintydev789** [comment]
If we're removing clause 7, then something equivalent should probably be added to the Bylaws, as point 3.8 in the Bylaws is more limited in scope than clause 7 here.

**Comment by EdHollingdale** [comment]
> If we're removing clause 7, then something equivalent should probably be added to the Bylaws, as point 3.8 in the Bylaws is more limited in scope than clause 7 here.

I think changing the bylaws to a particular system is a separate decision to this motions change. The point is to remove from the motions a system which does not match what is in the bylaws (since the bylaws supersede the motions, and having two systems causes confusion). 

I think the bylaws are pretty clear and, given I think the likelihood of this needing to be used is low, I don't think we need to spend energy perfecting the exact system, which currently is:
3.8: directors can be removed by members by a majority vote at a meeting of the organization that has quorum.
2.11: a meeting can be called via a petition signed by 5% of voting members
2.12: the meeting must be called with at least two weeks' notice
2.13: the meeting must be attended by at least 1/3rd of voting members. 

I would support a more general uplift of the bylaws at a future date which could include changing this, but I don't think it should be prioritised.

**Comment by Epride** [comment]
> If we're removing clause 7, then something equivalent should probably be added to the Bylaws, as point 3.8 in the Bylaws is more limited in scope than clause 7 here.

Are you able to explain what limitation exist in the Bylaws clause 3.8 that is covered by this?

**Comment by mintydev789** [comment]
@Epride actually, Ed's clarification about point 2 has erased my concerns. Lgtm so far.

**Comment by lgarron** [comment]
> I would support a more general uplift of the bylaws at a future date which could include changing this, but I don't think it should be prioritised.

I think it should be prioritized. In fact, I think it should be a blocker for merging this change to the Motions.

I understand that the Board is trying to be practical, but I am extremely skeptical about removing this protection before it's covered equally clearly the Bylaws. This kind of bureaucratic detail is how important protections can be lost, and the "vote of no confidence" is an important protection against egregious conduct by the Board.

I would hope that we don't have a Board that would go rogue any time soon, but if this ever happens then presumably the Board against anything that is not super clearly spelled out in the Bylaws. 3.8 of the current Bylaws is rather vague compared to the wording being removed in this PR, and I do not feel comfortable relying on it.

Just to put my cards on the table: If this change is merged without a preceding change in the Bylaws to spell out the same mechanism, I intend to argue strongly against this change to the WCA Board motions. I plan to vote against it and will advocate for others to do so as well.

**Comment by EdHollingdale** [comment]
Why is this change something that has an impact though Lucas? Given the motions currently contradict the bylaws the hierarchy of WCA documents means that this motion does not hold anyway, the mechanism in the bylaws is what applies.

I also fail to see what is unclear about the process in the bylaws. They allow for a mechanism to remove directors.

**Comment by lgarron** [comment]
> Why is this change something that has an impact though Lucas? Given the motions currently contradict the bylaws the hierarchy of WCA documents means that this motion does not hold anyway, the mechanism in the bylaws is what applies.
> 
> I also fail to see what is unclear about the process in the bylaws. They allow for a mechanism to remove directors.

Bylaws 3.8 state:

> A director may be removed from office at any time, with or without cause, by a vote of a majority of
directors then in office, or a vote of a majority of members at a properly announced meeting,
provided that a quorum is present.

This does not make clear:

- What constitutes a properly announced meeting? Can any voting member call one? Any WCA staff? Anyone at all? 2.12 only states that "Printed notice of each meeting shall be given to each voting member, by mail or electronic
correspondence, not less than two weeks prior to the meeting."
- Who conducts the vote and declares the result?
- Is it possible to vote on removing all Board members at once, or just one at a time? Voting on all members at once is much better for avoiding edge cases, but is not explicitly supported by 3.8.

These things wouldn't usually matter, if the WCA is functioning harmoniously. Similar language appears in the Bylaws where I find it less concerning. But these things can become really important if the voting members ever need to use this power.

**Pull request merged**